src/types/registration.types.ts

```diff
@@ -133,7 +133,7 @@ interface GeneralPublicationNote {
 type PublicationNote = UnpublishingNote | GeneralPublicationNote;
 
 export interface BaseRegistration {
-  readonly type: 'Publication' | 'ImportCandidate';
+  readonly type: 'Publication' | 'ImportCandidate' | 'PartialUpdatePublicationRequest';
```

Comment:
`PartialUpdatePublicationRequest` skulle trolig ikke vært her? 🤔 
`BaseRegistration` er en grunnbeskrivelse av Registration, mest for respons fra GET Publication til backend, men også PUT. Men så lenge `PartialUpdatePublicationRequest` bare skal brukes ved ett tilfelle og det er ved partial update tenker jeg det bare skulle vært i update-kallet (som det også er)